src/components/SendTokensOne.js

```diff
@@ -216,8 +193,8 @@ class SendTokensOne extends React.Component {
     const renderSelectToken = () => {
       return (
         <select className="ml-3" value={this.state.selected.uid} onChange={this.changeSelect}
-          title={hathorLib.wallet.isHardwareWallet() ? t`This feature is disabled for hardware wallet` : t`Select token`} 
-          disabled={hathorLib.wallet.isHardwareWallet() ? true : null}>
+          title={version.isLedgerCustomTokenAllowed() ? t`Select token` : t`This feature is disabled for hardware wallet`}
```

Comment:
Is this message `This feature is disabled for hardware wallet` still accurate? 

Shouldn't we change it to say it is disabled for the current ledger version?